e2e-tests/tests/block_login_with_different_username.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Resource        ./resources/authd/utils.resource
+Resource        ./resources/authd/authd.resource
+
+Resource        ./resources/broker/broker.resource
+
+Test Tags       robot:exit-on-failure
+
+Test Teardown    Log Videos On Error
+
+
+*** Variables ***
+${username}    %{E2E_USER}
+${local_password}    qwer1234
+${remote_group}    %{E2E_USER}-group
+
+
+*** Test Cases ***
+Log in with local user
+    Log In
+
+
+Fail to login if usernames do not match
+    Open Terminal
+    Start Log In With Remote User Through CLI: QR Code   different_user
+    Select Provider
+    Continue Log In With Remote User: Log In On External Browser   ${username}
+    Check That User Information Can Not Be Fetched
```

Comment:
This step fails because the error message that's displayed is `Authentication failure: requested username "different_user" does not match the authenticated username`, not `authentication failure: could not fetch user info` (I think I changed that in a recent PR). 

I'll a push a commit to fix that.

Reply:
done